css/style.css

```diff
@@ -9,45 +9,172 @@ Developer @GA in NYC
 /* SETUP
 /*******************************************/
 
-html {
```

Comment:
Why did you remove these setup styles? These are your box-sizing properties that are necessary and should be included with every stylesheet!